createdisk-library.sh

```diff
@@ -87,15 +91,18 @@ function create_qemu_image {
 
     if [ -f /var/lib/libvirt/images/${overlay} ]; then
       sudo cp /var/lib/libvirt/images/${overlay} ${destDir}
```

Comment:
I'd add a comment either here or before `local overlay=$3` to indicate that `${overlay}` won't exist in some cases, for example when using microshift.

Reply:
@cfergeau updated.